README.md

```diff
@@ -116,6 +116,7 @@ The API serves the following endpoints:
 *Related to checks:*
 - `GET` on `/api/checks/latest/?url={url}&resource_id={resource_id}` to get the latest check for a given URL and/or `resource_id`
 - `GET` on `/api/checks/all/?url={url}&resource_id={resource_id}` to get all checks for a given URL and/or `resource_id`
+- `GET` on `/api/checks/aggregate/?group_by={column}&created_at={date}` to get checks occurences grouped by a `column` for a specific `date`
```

Comment:
I guess {date} is supposed to be an time interval (since `today` is possible)? If yes, can it not only a day, but also an hour, a month or a year? Maybe it would be good to specify what's possible in terms of time format interval in the doc, and maybe even to have specific error responses in case of not accepted time interval.

Reply:
My personal use case was `today` (or latest) but it seemed fair and easy to allow to pick another date.
It is currently not a datetime interval since it's used as `created_at::date = $1`.

Making it an interval would probably require a `start` and `end` param I guess. Maybe we could have a start at `now - 1 day` and end at `now` to replicate the today (or other latest heuristic) default?

Reply:
I don't think we need to make it an interval with `start` and `end` params yet, what I meant is `today`is technically an interval (from midnight to midnight). So what other similar "one-word" interval it accepts and doesn't, should be documented. Does that make sense?

Reply:
Done in https://github.com/datagouv/hydra/pull/167/commits/2aba2ee4ed35889518a148ed3884faccfcbb64ba